libusb/libusbi.h

```diff
@@ -164,7 +164,7 @@ static inline void *usbi_reallocf(void *ptr, size_t size)
 #ifndef TIMESPEC_TO_TIMEVAL
 #define TIMESPEC_TO_TIMEVAL(tv, ts)                                     \
         do {                                                            \
-                (tv)->tv_sec = (ts)->tv_sec;                            \
+                (tv)->tv_sec = (long) (ts)->tv_sec;                            \
```

Comment:
long is 32-bit on Windows. time_t is 64-bit. timeval.tv_sec is long from winsock.h, it can't be changed.

There are definitely Y2038 problems, but other places are littered with `(long)` cast. Fixing Y2038 bugs would be of a future PR.